test/ModuleInterface/back-deploy-attr.swift

```diff
@@ -17,27 +17,24 @@
 
 public struct TopLevelStruct {
   // CHECK: @_backDeploy(macOS 11.0)
-  // CHECK: public func backDeployedFunc1_SinglePlatform() -> Swift.Int
+  // FROMSOURCE: public func backDeployedFunc1_SinglePlatform() -> Swift.Int { return 42 }
+  // FROMMODULE: public func backDeployedFunc1_SinglePlatform() -> Swift.Int
```

Comment:
Is it expected to lose the inlinable code when serialized?

I'm aware of some issues with losing classic `@inlinable` code with merge-module, that might explain it and may not be worth fixing in such a case. It was one source of broken swiftinterfaces when they were generated in incremental/merge-module mode.

Reply:
You'll see something similar in the `inlinable-functions.swift` test case; I'm not sure if it's correct behavior but it does seem like this is currently expected to happen.

Reply:
Interesting... I never looked closely at `inlinable-function.swift`. IIUC the way the test is set up, it's testing a behavior that's not applied in practice, it would be compiling a swiftinterface in the SDK to a cached swiftmodule, and then generating a swiftinterface for it from the cached swiftmodule. Back to this test, I think we should expect to keep the inlinable code after merge-module, but it's merge-module so we don't need to support it anymore.

Reply:
Maybe there's a better way to accomplish what I'm trying to accomplish with that aspect of the test, which is just to verify that the attribute can be deserialized from modules successfully. I think on a previous PR we talked about how this step is maybe unnecessary once there is another test case where there are separate client and library source files, compiled separately